content/tutorials/kit-assembly.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Connecting your kit
+---
+
+## The bits you should have
+- Raspberry Pi (the board in a plastic case with many USB ports)
+- Power Board (the board in a plastic case with many green sockets and two buttons)
+- Motor Board (the other board in a plastic case)
+- Arduino (The blue, exposed board with a metal USB-B connector)
+- Arduino Servo Shield (another blue board, hopefully on top of the first)
+- 1 x 3.81mm Camcon (for the Arduino Servo Shield)
+- 2 x 5mm Camcon (for the Motors)
+- 1 x 5mm Camcon with wire loop (attached to the power board)
+- 2 x 7.5mm Camcon (for the Motor Board)
+- USB Hub
+- USB Webcam
+- 2 x Black USB-A to micro-usb cables
+- Beige USB-A to USB-B cable (for the Arduino)
+- camcon to micro-usb cable (for the Raspberry Pi)
+- Pliers
```

Comment:
I don't think we've yet decided whether we plan to give out tools and/or motors to teams for SB2018, but I imagine it's unlikely.

I think it's valid to leave these in here for now, so long as we add a Trello card to update it once we've made a decision.

Reply:
Will card that up now